src/db/commit.rs

```diff
@@ -3,6 +3,8 @@ use crate::dag;
 use flatbuffers::FlatBufferBuilder;
 use str_macro::str;
 
+pub const DEFAULT_HEAD_NAME: &str = "main";
```

Comment:
I wasn't clear on this in my last review comment - my mistake. I think it makes more sense to leave this at the db layer and reference it from embed. I don't think Commit should know what the "default head name" is - it's a feature that it is ignorant of it. This is a micronit and I wasn't clear in my last review, so I will fix it myself, someday :).